lib/librte_ip_frag/rte_ip_frag_common.c

```diff
@@ -93,6 +94,31 @@ rte_ip_frag_table_destroy(struct rte_ip_frag_tbl *tbl)
 	rte_free(tbl);
 }
 
+/* Iterate over the IP fragmentation table. */
+int
+rte_ip_frag_table_iterate(struct rte_ip_frag_tbl *tbl,
+	struct rte_ip_frag_death_row *dr, uint32_t *next)
```

Comment:
This function is not an iterator per se because the caller doesn't have a way to access all the fragments in the table. What is really being done is sweeping the table for expired segments. So we should rename this function to `rte_ip_frag_sweep_table()`.